package.json

```
@@ -17,9 +17,11 @@
     "i18njson": "node tasks/i18njson",
     "update-deps": "rm npm-shrinkwrap.json && rm -rf node_modules && npm install && npm shrinkwrap",
     "build": "echo yes | npm run release && cp release/woocommerce-services.zip ./",
-    "docker:up": "npm run dist && npx wc-e2e docker:up",
-    "docker:down": "npx wc-e2e docker:down",
-    "docker:ping": "bash ./tests/bin/wait-for-e2e-build.sh",
+	"up": "docker-compose up --build --force-recreate -d && ./bin/docker-setup.sh",
+	"docker:down": "docker-compose down",
```

Just for consistency's sake, could we either use `npm run up` and `npm run down` or `npm run docker:up` and `npm run docker:down`? Already had to curse my terminal window a few times after seeing the `npm ERR! missing script` error flash across my screen because I picked the wrong one.

Personally, I prefer the shorter `npm run up/down`, because it's, well, shorter. But also, consistent with other containerised Woo plugins.

---

You're absolutely right, let's do the shorter version `up`/`down` 👍 
```suggestion
	"down": "docker-compose down",
```